docs/sql/mysql/create_schema.sql

```diff
@@ -173,7 +174,7 @@ CREATE TABLE da_sms_authorization (
   salt                 VARBINARY(16) NOT NULL,
   message_text         TEXT NOT NULL,
   verify_request_count INTEGER,
-  verified             BOOLEAN,
+  verified             BOOLEAN DEFAULT FALSE,
```

Comment:
Should we include those changes in db migration changes?
```
ALTER TABLE `ns_user_prefs` ALTER COLUMN `auth_method_1` SET DEFAULT FALSE;
ALTER TABLE `ns_user_prefs` ALTER COLUMN `auth_method_2` SET DEFAULT FALSE;
...
UPDATE `ns_user_prefs` SET `auth_method_1` = FALSE WHERE `auth_method_2` is null;
UPDATE `ns_user_prefs` SET `auth_method_2` = FALSE WHERE `auth_method_2` is null;
...
```

Reply:
I will add it.